ci/release/_install.sh

```diff
@@ -204,36 +236,61 @@ EOF
     log "  man d2plugin-tala"
   fi
   else
-    log "  Run man d2 for detailed docs."
+    log "Run man d2 for detailed docs."
     if [ -n "${TALA-}" ]; then
-      log "  Run man d2plugin-tala for detailed docs."
+      log "Run man d2plugin-tala for detailed TALA docs."
     fi
   fi
   logcat >&2 <<EOF
 
 Something not working? Please let us know:
-https://github.com/terrastruct/d2/issues/new
+https://github.com/terrastruct/d2/issues
+https://github.com/terrastruct/d2/discussions
+https://discord.gg/NF6X8K4eDq
```

Comment:
kind of overwhelming. just issues i'd say

Reply:
Can we just remove this please, it's just noise.

Reply:
i'm like an 8/10 here that these add value, which is pretty strong but not full insist. if you're higher in your opinion, ok we'll remove.

Reply:
Ok, let's leave these but then also always print next steps.